include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -213,7 +213,7 @@ struct transform_reduce
     void
     vectorized_reduction_first(const _Size __start_idx, _Res& __res, const _Acc&... __acc) const
     {
-        new (&__res.__v) _Tp(__unary_op(__start_idx, __acc...));
+        __res.__setup(_Tp(__unary_op(__start_idx, __acc...)));
```

Comment:
Do we need the `_Tp()` here?  I think that should be redundant.

Reply:
You are correct; they are redundant and have been removed.